cns/networkcontainers/networkcontainers_linux.go

```diff
@@ -13,10 +21,65 @@ func setWeakHostOnInterface(ipAddress string) error {
 	return nil
 }
 
-func createOrUpdateWithOperation(createNetworkContainerRequest cns.CreateNetworkContainerRequest, operation string) error {
+func updateInterface(createNetworkContainerRequest cns.CreateNetworkContainerRequest, netpluginConfig *NetPluginConfiguration) error {
+	log.Printf("[Azure CNS] update interface operation called.")
+
+	// Currently update via CNI is only supported for ACI type
+	if createNetworkContainerRequest.NetworkContainerType != cns.AzureContainerInstance {
+		log.Printf("[Azure CNS] operation is only supported for AzureContainerInstance types.")
+		return nil
+	}
+
+	if _, err := os.Stat(netpluginConfig.path); err != nil {
```

Comment:
check if netpluginconfig is nil before dereferencing
if netpluginConfig == nil {
    log(errmsg) 
    return err
}